lib/core/src/lib/datatable/components/datatable/datatable.component.html

```diff
@@ -125,14 +126,15 @@
             >
                 <ng-container *ngIf="mainActionTemplate">
                     <button
+                        class="adf-datatable__actions-cell-button"
```

Comment:
is that class needed? I can't see styles for it

Reply:
good catch, removed